react/components/ProductAssemblyOptions/ProductAssemblyOptionsGroup.test.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { render, fireEvent } from '@vtex/test-tools/react'
+import { render, fireEvent, within } from '@vtex/test-tools/react'
```

Comment:
Just out of curiosity, what's this `within`?

Reply:
This: https://testing-library.com/docs/dom-testing-library/api-helpers#within-and-getqueriesforelement-apis